driver-sync/src/test/functional/com/mongodb/client/unified/ContextElement.java

```diff
@@ -73,6 +73,74 @@ public static ContextElement ofConnectionPoolEvent(final BsonDocument expected,
         return new ConnectionPoolEventMatchingContextElement(expected, actual, eventPosition);
     }
 
+    public static ContextElement ofWaitForPrimaryChange() {
```

Comment:
New operations, new contexts.  Anonymous classes seem fine here, though I don't remember why that wasn't done for many of the existing ones.